modules/react/loading-dots/stories/LoadingDots.mdx

```diff
@@ -29,6 +30,10 @@ yarn add @workday/canvas-kit-react
 
 <ExampleCodeBlock code={RTL} />
 
+### Circle Variant
```

Comment:
Let me know what you think of this naming convention @mannycarrera4 ! Rather than calling out the ability to change the background color of the dots explicitly I felt like using the "Circle Variant" name matches our design docs a bit more closely 
https://canvas.workdaydesign.com/components/indicators/loading-dots/

Reply:
nice that makes sense thanks for adding that!